api/v1/server/embedded_spec.go

```diff
@@ -1709,6 +1709,25 @@ func init() {
         }
       }
     },
+    "PolicyRule": {
+      "description": "A policy rule including the rule labels it derives from",
+      "properties": {
+        "derived-from-rules": {
+          "description": "The policy rule labels identifying the policy rules this rule derives from",
+          "type": "array",
+          "items": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        },
+        "rule": {
+          "description": "The policy rule as json",
```

Comment:
I totally missed this but I can fix it up in another PR. It isn't terribly useful as a description, but I'm not sure I understand what you mean by "what the rule will appear as". This type replaces what was a string array with the description "List of L4 ingress rules" (or L3). Did you something like this in mind?

The policy rule's metadata and L3/L4/L7 restrictions as derived from the union of the set of rules in the cilium agent

(I was trying to avoid using the word rule again, although This derived rule might not be 1:1 equivalent to something submitted by the user).